charts/enterprise-kyverno-operator-crd/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: nirmata-kyverno-operator-crd
 description: Helm Chart for Enterprise Kyverno Operator
 type: application
-version: v0.4.7
+version: v0.4.8
```

Comment:
do we need to bump version for README.md changes?

Reply:
lint tests were failing without bumping

Reply:
got it